packages/app/src/files-manager/components/files-summary-item/files-summary-item.component.tsx

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import React from "react"
+import Text, {
+  TextDisplayStyle,
+} from "Renderer/components/core/text/text.component"
+import { DiskSpaceCategory } from "App/files-manager/components/files-manager/files-manager.component"
+import { backgroundColor } from "Renderer/styles/theming/theme-getters"
+import { FunctionComponent } from "Renderer/types/function-component.interface"
+import styled from "styled-components"
+import Icon, { IconSize } from "Renderer/components/core/icon/icon.component"
+import { FilesSummaryItemTestIds } from "App/files-manager/components/files-summary-item/files-summary-item-test-ids.enum"
+import { displaySpace } from "App/files-manager/helpers/display-space"
+
+const Wrapper = styled.div`
+  display: flex;
+  align-items: flex-start;
+  background-color: ${backgroundColor("row")};
+  padding: 1.6rem;
+  width: 28rem;
+  :not(:last-of-type) {
+    margin-right: 3.2rem;
```

Comment:
🟡
In my opinion, the parent component should determine the style of the child. Item component should a raw like is possible. So I think that the margin style set could be moved higher. The margin style is important from the list perspective no item.

Reply:
Ok, moved in [42dd729](https://github.com/mudita/mudita-center/pull/598/commits/42dd7294f4da56efac333877d32fd5442ff68173)